app/src/main/java/org/oppia/app/profile/AdminAuthActivity.kt

```diff
@@ -14,10 +14,11 @@ class AdminAuthActivity : InjectableAppCompatActivity() {
   lateinit var adminAuthFragmentPresenter: AdminAuthActivityPresenter
 
   companion object {
-    fun createAdminAuthActivityIntent(context: Context, adminPin: String): Intent {
+    fun createAdminAuthActivityIntent(context: Context, adminPin: String, color: Int): Intent {
```

Comment:
Ditto here & elsewhere in the PR: please make the type of ``color`` evident. From your use above, it seems like it's a resource ID.